ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -499,7 +500,7 @@ public void commit() {
       } catch (Exception e) {
         LOG.warn("{} Commit optimizing failed ", tableRuntime.getTableIdentifier(), e);
         status = Status.FAILED;
-        failedReason = e.getMessage();
+        failedReason = ExceptionUtil.getErrorMessage(e, 4000);
```

Comment:
Can you append some errors from up level? For example: UnKeyedTableCommit#replaceFiles?

Reply:
For the convenience of troubleshooting, I have displayed as much information as possible here, including the exception stack.

Although it may appear a bit verbose on the current page, these exception details will be moved to the subsequent process detail page.